lib/matplotlib/axes/_axes.py

```diff
@@ -5517,9 +5517,11 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
             which can be set by *filterrad*. Additionally, the antigrain image
             resize filter is controlled by the parameter *filternorm*.
 
-        alpha : scalar, optional
+        alpha : [scalar | array_like], optional, default: None
```

Comment:
`scalar or array-like`

Reply:
`float or array-like` according to our current [doc conventions](https://matplotlib.org/devel/documenting_mpl.html?highlight=documenting#parameter-type-descriptions).

Reply:
I'll keep this as `scalar` for consistency with all other docstrings in `_axes.py`.